content/en/basics/interplanetary-consensus/mir-and-trantor/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Mir and Trantor"
+description: "Alongside the scaling framework, the IPC project is developing high-performance consensus protocols that are suitable for subnet use."
+lead: "Alongside the scaling framework, the IPC project is developing high-performance consensus protocols that are suitable for subnet use."
+draft: false
+images: []
+type: docs
+menu:
+  basics:
+    parent: "basics-interplanetary-consensus"
+    identifier: "ipc-mir-and-trantor"
+weight: 120
+toc: true
+---
+
+Beyond their function as a scaling tool, IPC subnets will be deployed to meet the requirements of applications that cannot be run on the Filecoin rootnet. The combination of Mir and Trantor provides a high-performance consensus mechanism with faster finality and higher throughput. Moreover, Trantor can be further configured for each subnet.
+
+## Mir framework
+
+Mir is a framework for implementing, debugging, and analyzing distributed protocols. It takes the form of a library that provides abstractions representing different components of a distributed system and an engine orchestrating their interaction.
+
+Mir aims to be general enough to enable implementing distributed protocols in a way that is agnostic to network transport, storage, and cryptographic primitive implementation. All these (and other) usual components of a distributed protocol implementation are encapsulated in abstractions with well-defined interfaces. While Mir provides some implementations of those abstractions to be used directly "out of the box", the consumer is free to provide their own implementations.
+
+The first use of Mir is as a scalable and efficient consensus layer in Filecoin subnets. Learn more on [GitHub](https://github.com/filecoin-project/mir).
+
```

Comment:
```suggestion
The goal of Mir is to enable implementation of distributed protocols in a way that is agnostic to network transport, storage, and cryptographic primitive implementation. All of these components of a distributed protocol implementation, as well as others, are encapsulated in abstractions with well-defined interfaces. While Mir provides some implementations of abstractions  for direct use "out of the box", the user is free to define their own implementations.

Mir is currently being used as a scalable and efficient consensus layer in Filecoin subnets. Learn more at [GitHub](https://github.com/filecoin-project/mir).
```

Reply:
I like what you did but would further change the final sentence to

> Mir is used as a scalable and efficient consensus layer in IPC subnets.

Reply:
```suggestion
The goal of Mir is to enable implementation of distributed protocols in a way that is agnostic to network transport, storage, and cryptographic primitive implementation. All of these components of a distributed protocol implementation, as well as others, are encapsulated in abstractions with well-defined interfaces. While Mir provides some implementations of abstractions  for direct use "out of the box", the user is free to define their own implementations.

Mir is used as a scalable and efficient consensus layer in IPC subnets.. Learn more at [GitHub](https://github.com/filecoin-project/mir).
```
(your suggestion with tweaks)

I like what you did but would further change that last paragraph. Reasoning: currently suggests transient, which isn't the case, and "Filecoin subnets" is poorly defined (mea culpa).